app/src/androidTest/java/com/mycompany/myapp/RecyclerViewMatcher.kt

```diff
@@ -43,7 +43,7 @@ class RecyclerViewMatcher(private val recyclerViewId: Int) {
                 this.resources = view.resources
 
                 if (childView == null) {
-                    val recyclerView = view.rootView.findViewById<View>(recyclerViewId) as RecyclerView
+                    val recyclerView = view.rootView.findViewById<View>(recyclerViewId) as androidx.recyclerview.widget.RecyclerView
```

Comment:
Can we fix the FQN (fully qualified names) of the classes throughout? The import should update and probably not the rest of the code.